File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.java
import java.util.Arrays;
class Solution {
    public String[] solution(String[] quiz) {
      	
        String[] quizarr = new String[quiz.length];
		String[] answer = new String[quiz.length];
		
		for(int i=0; i<quiz.length; i++) {
			
			quizarr = quiz[i].split(" ");
				
				if(quizarr[1].equals("-")) {
					if(Integer.valueOf(quizarr[0]) - Integer.valueOf(quizarr[2]) == Integer.valueOf(quizarr[4])) {
						answer[i] = "O";
					}else {
						answer[i] = "X";
					}
				}
				if(quizarr[1].equals("+")) {
					if(Integer.valueOf(quizarr[0]) + Integer.valueOf(quizarr[2]) == Integer.valueOf(quizarr[4])) {
						answer[i] = "O";
					}else {
						answer[i] = "X";
					}
				}
				
			}

			return answer;
		}
    
}

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    li = [i.split(' = ') for i in quiz]
    answer = []
    for i in li:
        if ' - ' in i[0]:
            tmp_li = i[0].split(' - ')
            if int(i[1]) == (int(tmp_li[0]) - int(tmp_li[1])):
                answer.append('O')
            else:
                answer.append('X')
        elif ' + ' in i[0]:
            if int(i[1]) == sum(int(j) for j in i[0].split(' + ')):
                answer.append('O')
            else:
                answer.append('X')
    return answer